esx_service/utils/auth.py

```diff
@@ -34,6 +34,7 @@
 
 SIZE = 'size'
 DEFAULT_TENANT = '_DEFAULT'
+DEFAULT_TENANT_UUID = '_DEFAULT_TENANT_UUID'
```

Comment:
If team accepts a proposal to rename TENANT => Namespace then folder _DEFAULT_TENANT_UUID will out of place. I propose we pick a magic string. Also test if randomly generated UUID happen to match with magic string then collisions are handled correctly.